grafana/grafana
# GrafanaDS: Add support for annotation time regions

<img width="1715" alt="time-regions1" src="https://user-images.githubusercontent.com/88068998/233487642-2f59f055-c57d-4235-99da-f0879ebb66dc.png">

Setting up time regions:


https://user-images.githubusercontent.com/88068998/233487924-b7db6760-fbe1-44f1-941e-4d84501a5aa4.mov



Fixes: #35486


Testing steps:

1. Create a Dashboard
2. Add a Timeseries panel
3. Go to Dashboard Settings -> Annotations -> New Query
4. Add a new annotation with Data source: -- Grafana --
5. Query section - choose Query type: Time regions
6. Set a region (from/to/time/timezone)
7. Apply/Save Dashboard
8. Go back to the panel, refresh, time regions annotations should be visible and correctly represented
